lib/apiservers/service/swagger.json

```diff
@@ -100,6 +100,26 @@
         }
       }
     },
+    "/target/{target}/vch/{vchId}/certificate": {
```

Comment:
I think you'll also want to define an endpoint for `"/target/{target}/datacenter/{datacenter}/vch/{vchId}/certificate"`.